include/json/value.h

```diff
@@ -585,6 +585,22 @@ class JSON_API Value {
   iterator begin();
   iterator end();
 
+  /// @brief Returns a reference to the first element in the container.
+  /// Calling front on an empty container is undefined behavior.
```

Comment:
I think the term "container" can be confusing?
Instead of saying what's undefined behavior, maybe state as a precondition.

"Requires that value holds an array or json object, with `size() > 0`."

Reply:
I changed it to `Value` instead. Do you think that fits better?
Also changed the explanation to your suggested one.